addons/stock/tests/test_stock_flow.py

```diff
@@ -1978,6 +1978,26 @@ def test_picking_scheduled_date_readonlyness(self):
 
         self.assertEqual(f.state, 'confirmed')
 
+    def test_stock_move_with_partner_id(self):
+        """ Ensure that the partner_id of the picking entry is
+        transmitted to the SM upon object creation.
+        """
+        partner = self.env['res.partner'].create({'name': 'Hubert Bonisseur de la Bath'})
+        product = self.env['product.product'].create({'name': 'Un petit coup de polish', 'type': 'product'})
+        wh = self.env['stock.warehouse'].search([('company_id', '=', self.env.company.id)], limit=1)
+
+        f = Form(self.env['stock.picking'], view='stock.view_picking_form')
```

Comment:
Just for your information (because it's clearly a detail), defining `view` is not necessary in this case